data-prepper-expression/src/main/antlr/DataPrepperExpression.g4

```diff
@@ -339,6 +340,7 @@ EXPONENTLETTER
     ;
 
 INTEGER: 'integer';
+BIG_DECIMAL: 'big_decimal';
```

Comment:
Didn't we use the term `decimal` to be consistent with OpenSearch?

Reply:
We did use `big_decimal`, but that was already released. We should fix this by adding an alias in another PR.